src/components/views/rooms/AuxPanel.js

```diff
@@ -82,6 +100,41 @@ module.exports = React.createClass({
         ev.preventDefault();
     },
 
+    _rateLimitedUpdate: new RateLimitedFunc(function() {
+        this.setState({counters: this._computeCounters()})
```

Comment:
I think it would be nice to do a bit less work here by checking if the feature is enabled before calling `setState` here. Although React will skip a re-render when the feature is off by calling `shouldComponentUpdate` (which will determine that the state is unchanged), it seems better to not even try updating state when the feature is off. It's also a bit clearer to future readers, I think.